lib/broadside/configuration/deploy_config.rb

```diff
@@ -111,22 +115,33 @@ def load_env_vars!
 
       private
 
-      def self.validate_types(types, target_attribute)
-        if types.include?(target_attribute.class)
-          nil
-        else
-          "'#{target_attribute}' must be of type [#{types.join('|')}], got '#{target_attribute.class}' !"
+      class << self
+        def validate_types(types, target_attribute)
+          if types.include?(target_attribute.class)
+            nil
+          else
+            "'#{target_attribute}' must be of type [#{types.join('|')}], got '#{target_attribute.class}' !"
+          end
+        end
+
+        def validate_predeploy_commands(commands)
+          validate_commands(commands, 'predeploy_commands')
         end
-      end
 
-      def self.validate_predeploy_commands(commands)
-        return nil if commands.nil?
-        return 'predeploy_commands must be an array' unless commands.is_a?(Array)
+        def validate_bootstrap_commands(commands)
```

Comment:
v. minor but couldn't we just call `validate_commands(commands, 'bootstrap_commands')` instead of having 2 methods?